abel/linbasex.py

```diff
@@ -235,7 +238,7 @@ def _linbasex_transform_with_basis(IM, Basis, proj_angles=[0, np.pi/2],
 
     radial = np.linspace(clip, cols//2, len(Beta[0]))
 
-    return inv_IM, radial, Beta, QLz
+    return inv_IM*2/cols, radial, Beta, QLz
```

Comment:
I'm not sure if this is really the correct scaling factor after all. Perhaps add a comment that we are not yet sure of the correct scaling factor.

Reply:
Ok, checking this. Note, the image scaling will have no effect on the radial intensity.

Reply:
You are correct. `2/cols` works for `O2-ANU1024.txt` and `O2-ANU2048.txt`, but not `Sample.Image(n)`:
![linvs3pt](https://user-images.githubusercontent.com/10932229/34509794-09f44aa4-f0a2-11e7-86d6-5eab408af1a6.png)
[code gist](https://gist.github.com/stggh/2f898afb902867f26cdf29ba48c9d0f7)

I have removed the factor and added a comment referencing issue #202

Reply:
Okay, fine to leave this issue for another day. Can you open an Issue saying that we still need to figure out the scaling for Lin-BASEX?